packages/apps/kadena-docs/src/pages/docs/kadena/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Overview of Kadena
+menu: Kadena
+label: Introduction
+order: 0
+description: Kadena makes blockchain work for everyone.
+layout: full
+---
+
+### header
+
+sdfsdf
```

Comment:
Is this a left over from a test? :stuck_out_tongue_closed_eyes:

Reply:
its just test text